src/sharedData/components/LinkIcon.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import { SvgIcon } from '@mui/material';
+
+const LinkIcon = props => (
```

Comment:
question: Why do we have an inline SVG? If this is not a standard MUI icon, put int in src/assets (and run it through SVGOMG).

Reply:
@josebui I optimized it using SVGOMG, I am not sure about moving it to src/assets, this is how MUI suggest creating svg icons, so this makes seems like a proper solution https://mui.com/material-ui/icons/#svgicon

Reply:
Can you read the contents of the SVG (in src/assets) with `FileReader` or similar?

Reply:
@paulschreiber Maybe, yes, I would like to understand the why behind the sugestion, why do you think that is better? and why do you think the current approach is not enough?

Reply:
In general, it is a good practice to store assets outside of code, where that can be easily viewed/edited.